core/utils/poly-look/src/react/chips/chips.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import "./chips.css";
+
+const Chips = ({ chipsContent, activeChip, onChipClick, theme = "light" }) => {
+  return (
+    <div className={`poly-theme-${theme} chips-container`}>
+      {chipsContent.map(({ id, translation }) => {
```

Comment:
(coming from :arrow_down: ) maybe call this `title` instead of `id` to account for the case when translation does not exist?

Reply:
Yeah title is good :)
As you said it's for when it doesn't use translations eg. when it's numbers